packages/models-library/src/models_library/api_schemas_storage/search_async_jobs.py

```diff
@@ -0,0 +1,12 @@
+import datetime
+
+from models_library.projects import ProjectID
+from pydantic import BaseModel
+
+
+class SearchResult(BaseModel):
```

Comment:
is this going out in this format or there is an OutputSchema transforming it to Javascript style?